Userland/Utilities/run-tests.cpp

```diff
@@ -322,16 +328,24 @@ int main(int argc, char** argv)
     if (config->num_groups() == 0)
         warnln("Empty configuration file ({}) loaded!", config_file.is_empty() ? "User config for Tests" : config_file.characters());
 
-    if (exclude_pattern.is_empty())
+    if (exclude_pattern.is_empty()) {
         exclude_pattern = config->read_entry("Global", "NotTestsPattern", "$^"); // default is match nothing (aka match end then beginning)
-
+    }
     Regex<PosixExtended> exclude_regex(exclude_pattern, {});
     if (exclude_regex.parser_result.error != Error::NoError) {
         warnln("Exclude pattern \"{}\" is invalid", exclude_pattern);
         return 1;
     }
 
-    TestRunner test_runner(test_root, move(exclude_regex), move(config), print_progress, print_json, print_all_output);
+    // we need to preconfigure this, because we can't autoinitialize Regex types
+    // in the Testrunner
+    StringBuilder builder;
+    builder.join("|", config->read_entry("Global", "SkipRegex", "$^").split(' '));
```

Comment:
This seems a little odd, splitting regular expressions with space could make them invalid, for instance: `SomeRandomTest-[^ ]+` would be transformed to `SomeRandomTest-[^|]+` with this.

Reply:
So I just take a patten instead